### PR TITLE
add ignore_ssl_errors option

### PR DESCRIPTION
**Pull request by mtanda**
I added `ignore_ssl_errors` option which is already marked as TODO.

### CODE REVIEW

File: uptimerobot/resource_uptimerobot_monitor_test.go [hunk 1]
```diff
@@ -211,6 +211,39 @@ func TestUptimeRobotDataResourceMonitor_custom_port_monitor(t *testing.T) {
...
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMonitorDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: fmt.Sprintf(`
+				resource "uptimerobot_monitor" "test" {
+					friendly_name     = "%s"
+					type              = "%s"
+					url               = "%s"
+					ignore_ssl_errors = 1
```
Comment (louy) [inline uptimerobot/resource_uptimerobot_monitor_test.go:229]: Shouldn't this be a Boolean?

File: uptimerobot/resource_uptimerobot_monitor_test.go [hunk 2]
```diff
@@ -211,6 +211,39 @@ func TestUptimeRobotDataResourceMonitor_custom_port_monitor(t *testing.T) {
...
+				}
+				`, FriendlyName, Type, URL),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "friendly_name", FriendlyName),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "type", Type),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "url", URL),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "ignore_ssl_errors", "1"),
```
Comment (louy) [inline uptimerobot/resource_uptimerobot_monitor_test.go:236]: Shouldn't this be a Boolean?

**Review by louy — COMMENTED** [review]
Minor comment otherwise looking good 👍

File: uptimerobot/resource_uptimerobot_monitor_test.go (hunk 1 above)
Comment (mtanda) [reply uptimerobot/resource_uptimerobot_monitor_test.go]: That's right. I fixed it.

File: uptimerobot/resource_uptimerobot_monitor_test.go (hunk 2 above)
Comment (mtanda) [reply uptimerobot/resource_uptimerobot_monitor_test.go]: fixed too.

**Comment by mtanda** [comment]
thanks!

**Pull request merged**